auth/auth-server/src/server/handle_external_match/mod.rs

```diff
@@ -247,7 +240,15 @@ impl Server {
         }
 
         // Record metrics
-        record_external_match_metrics(&order, match_resp.match_bundle, &labels, did_settle).await?;
+        record_external_match_metrics(&order, &match_resp.match_bundle, &labels, did_settle)
+            .await?;
+
+        // Record quote comparisons before settlement, if enabled
+        if let Some(quote_metrics) = &self.quote_metrics {
+            quote_metrics
+                .record_quote_comparison(&match_resp.match_bundle, labels.as_slice())
+                .await;
```

Comment:
We intentionally record quote comparison metrics before awaiting settlement to compare against alternative quotes as early as possible for accuracy, as solvers are likely comparing quotes as soon as they receive them. Can we keep the ordering but not panic on errors within `record_quote_comparison`?

Reply:
yup ack, my subsequent fixes allow for this, will do